.github/workflows/vrt.yml

```diff
@@ -1,6 +1,9 @@
 name: Visual regression testing
 
 on:
+    push:
+      branches:
+        - main
```

Comment:
We should probably see if we can fine tune this to ignore `release` commits 🤔

Reply:
I don't think it hurts to try this and see if releases are causing noise. I think releases won't trigger Chromatic because they don't alter files that chromatic is watching. We'll have to keep an eye on that to prove out that theory.

Reply:
You're probably right! I bet we'll know soon enough 😆